google/oauth2/credentials.py

```diff
@@ -292,3 +293,50 @@ def to_json(self, strip=None):
             prep = {k: v for k, v in prep.items() if k not in strip}
 
         return json.dumps(prep)
+
+
+class UserAccessTokenCredentials(credentials.Credentials):
+    """Access token credentials for user account.
+
+    Obtain the access token for a given user account or the current active
+    usr account with the `gcloud auth print-access-token` command.
+
+    Args:
+        account (Optional[str]): Account to get the access token for. If not
+            specified, the current active account will be used.
+    """
+
+    def __init__(self, account=None):
+        super(UserAccessTokenCredentials, self).__init__()
+        self._account = account
+
+    def with_account(self, account):
+        """Create a new instance with the given account.
+
+        Args:
+            account (str): Account to get the access token for.
+
+        Returns:
+            google.oauth2.credentials.UserAccessTokenCredentials: The created
+                credentials with the given account.
+        """
+        return self.__class__(account=account)
+
+    def refresh(self, request):
```

Comment:
Is it possible to default this to None, or does this break something?
```suggestion
    def refresh(self, request=None):
```

Reply:
this function override the abstract method defined in the [base class](https://github.com/googleapis/google-auth-library-python/blob/master/google/auth/credentials.py#L79), so `request` cannot be default to None.